packages/headless/src/features/question-answering/question-answering-analytics-actions.ts

```diff
@@ -301,53 +322,76 @@ export const logOpenSmartSnippetSuggestionSource = (
         }
       );
     },
-    analyticsType: 'itemClick',
-    analyticsPayloadBuilder: (state): ItemClick => {
+    analyticsType: 'Qna.CitationClick',
+    analyticsPayloadBuilder: (state): Qna.CitationClick => {
       const relatedQuestion = relatedQuestionSelector(
         state,
         payload.questionAnswerId
       );
-      const result = answerSourceSelector(state, relatedQuestion?.documentId)!;
-      const information = partialDocumentInformation(result, state);
       return {
-        searchUid: state.search?.response.searchUid ?? '',
-        position: information.documentPosition,
-        itemMetadata: analyticsEventItemMetadata(result, state),
+        answer: {
+          responseId: state.search?.response.searchUid || '',
```

Comment:
@jstpierref 

When a smart snippet suggestion source is clicked, you can see here that we are sending a `Qna.CitationClick` with the `searchUid` as the `responseId` field, so we have the identifier of the search query that was made.

However with one search query multiple smart snippet suggestions could have been returned, so we can't know which smart snippet suggestion  this event is related to. is that okay? what do you think?

Reply:
Ideally, what uniquely identify the citation would be the `citation.id`. Now, for Smart Snippets specific `permanentId` per citation (which is not unique), but I think this is good enough given the reporting needs.

Reply:
If both events become Qna.CitationClick what would be the way to differentiate the two events?
Differentiate a CitationClick from SmartSnippet from a CitationClick from RGA?

Reply:
The `answer.type` sent in the payload of the event will do this differentiation, it can take the following values: `RGA`,  `SmartSnippetSuggestion` or `SmartSnippet`